<launch>
  <node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
    <param name="width" value="320" type="int"/>
    <param name="height" value="240" type="int"/>
    <param name="fps" value="30" type="int"/>
    <param name="frame" value="wide_stereo" type="string"/>

    <param name="auto_focus" value="false" type="bool"/>
    <param name="focus_absolute" value="0" type="int"/>
    <!-- other supported params: auto_exposure, exposure_absolute, brightness, power_line_frequency -->

    <param name="device" value="/dev/video0" type="string"/>
    <param name="camera_info_url" value="file://$(find uvc_camera)/example.yaml" type="string"/>
  </node>
</launch>
